firebase-app-distribution/src/main/java/com/google/firebase/appdistribution/FirebaseAppDistribution.java

```diff
@@ -70,6 +70,11 @@
   private CancellationTokenSource checkForUpdateCancellationSource;
   private final Executor checkForUpdateExecutor;
 
+  private TaskCompletionSource<UpdateState> updateAppTaskCompletionSource = null;
+  private CancellationTokenSource updateAppCancellationSource;
+
```

Comment:
I would rename this to be more meaningful. Maybe `cachedLatestRelease` or `currentLatestRelease`?

Reply:
Open to other names too :) but appDistributionReleaseInternal is kinda ambiguous